tests/ui/issues/issue-15734.rs

```diff
@@ -1,6 +1,6 @@
 // run-pass
-// If `Index` used an associated type for its output, this test would
```

Comment:
`Index` does use an associated type in the ... 8? years since this test was written :grin: